docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

```diff
@@ -773,7 +788,7 @@ To split tests in the most efficient way possible, use:
 
 - `spec` when the number of specs is less than the configured concurrency.
 - `concurrency` when the number of specs is larger than the configured concurrency.
-:::
+  :::
```

Comment:
Formatter does it?
 
```suggestion
:::
```

Reply:
Interesting, yeah! But not for all, just for the notes that end with a bullet list. Strange 🤔